components/nav.vue

```diff
@@ -34,9 +34,14 @@
                 <a :href="utm('https://github.com/cdnjs', 'nav')">
                     GitHub
                 </a>
-                <a :href="utm('https://status.cdnjs.com', 'nav')">
-                    Status <StatusIndicator page-id="1fkb7yl9sw87"></StatusIndicator>
-                </a>
+                <div class="fix-mobile">
+                    <a href="javascript:void(0);" class="icon-mobile" @click="openMenu">
```

Comment:
```suggestion
                    <a class="icon-mobile" @click="toggleMenu" aria-label="Toggle menu">
```
I don't think we need `href` here + add an `aria-label` for screenreaders + let's call the method toggle.

Reply:
I really liked the solution by removing `javascript:void(0)` inside the `href`